tests/flow_generic/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -22,7 +22,16 @@ var X = {
     F,
     G,
     T: (a: A, b: B, c: C, d: D, e: E, f: F) => G // eslint-disable-line space-before-function-paren
-  >(method: T, scope: any, a: A, b: B, c: C, d: D, e: E, f: F): G {}
```

Comment:
Hey, I'm sorry but this is a regression as it was fitting in one line and now it doesn't anymore. There may be a `group()` that is no longer there that needs to be brought back.

I'm very sensitive to those things as they tend to introduce a lot of changes on real codebases.